sw/device/tests/aon_timer_wdog_bite_reset_test.c

```diff
@@ -19,7 +19,7 @@
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
-#define WDOG_BARK_TIME (5 * 1000)  // 5ms
+#define WDOG_BARK_TIME (200)  // 200 microseconds
```

Comment:
This is only used in two places. Inline it, or more it closer to where it is used?